src/Adapter/MSTest.TestAdapter/RunConfigurationSettings.cs

```diff
@@ -2,10 +2,14 @@
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
 using System.Diagnostics.CodeAnalysis;
-using System.Globalization;
 using System.Xml;
 
+#if !WINDOWS_UWP
+using System.Globalization;
+
```

Comment:
nit: empty line,

Reply:
We have policy to have empty line between groups of usings hence the space. Thanks @fhnaseer